src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/NotifichePersonaFisicaPagoPATest.java

```diff
@@ -341,6 +343,20 @@ public void siControllaLaPaginazioneDiDefault() {
         piattaformaNotifichePage.checkDefaultPagination();
     }
 
+    @And("Si controlla la pagina delle notifiche delegati di {string}")
+    public void siControllaLaPaginaDelleNotificheDelegatiDi(String nomeDelegante) {
+        piattaformaNotifichePage.checkPaginaNotificheDelegante(nomeDelegante);
+    }
+
+    @And("Si clicca sulla notifica del delegante")
+    public void siCliccaSullaNotificaDelDelegante() {
+        destinatarioPage.clickSulDettaglioNotificaDelegante();
+    }
+
+    @And("Si controlla il dettaglio della notifica")
+    public void siControllaIlDettaglioDellaNotifica() {
+        destinatarioPage.checkDettaglioNotificaPage();
+    }
```

Comment:
Have you check if there is other feature steps similar to what you're trying to accomplish?

Reply:
yes and deleted not used function